iOSClient/AppDelegate.swift

```diff
@@ -163,6 +163,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
                 }
             }
         }
+        
```

Comment:
This can be moved to a function in separate or new class to have single line of customization here.